Added Background music

FrederikSchlemmer opened the pull request:
Fixes #32 

Added background music and linked the source in the wiki.

FrederikSchlemmer committed:
Added background music

FrederikSchlemmer committed:
Changed the track

Psychedelic royalty free music from https://www.bensound.com/

RudiKlassen-zz commented on core/src/de/adesso/brainysnake/BrainySnake.java:
```diff
@@ -47,6 +54,12 @@ public void create() {
 
+        try {
+            backgroundSound = Gdx.audio.newMusic(Gdx.files.internal("./core/src/de/adesso/brainysnake/sound/track.mp3"));
+        } catch (Exception e){
```
Pls only check specific Exceptions. If this loc would throw an Exception the Stacktrace would be logged anyway so the catch can be deleted.

FrederikSchlemmer replied:
Deleted the try/catch block and the logger.

RudiKlassen-zz commented on core/src/de/adesso/brainysnake/BrainySnake.java:
```diff
@@ -102,6 +115,11 @@ private void initializeCamera() {
     }
 
     public void drawStartScreen() {
+        if(!backgroundSound.isPlaying()){
```
there is  no need to check this every frame. Pls move this to an own method and only call it within the create() method.

FrederikSchlemmer replied:
Created an own method to start music, which is once called after the initialization.
Maybe the if-statement is unnecessary at this point. Please check if it's useful for protective reasons to prevent playing same music in multiple loops.

FrederikSchlemmer committed:
Created own method

Created own method for starting the background music.
Deleted the try/catch block at initializing the music.
Added dispose method to the background music.

Based on the wiki :https://github.com/libgdx/libgdx/wiki/Streaming-music

FrederikSchlemmer committed:
Deleted unused logger

FlorianTim commented on core/src/de/adesso/brainysnake/BrainySnake.java:
```diff
@@ -47,6 +54,12 @@ public void create() {
         texture = new Texture(pixmap);
 
+        try {
+            backgroundSound = Gdx.audio.newMusic(Gdx.files.internal("./core/src/de/adesso/brainysnake/sound/track.mp3"));
```
Please move the audio file to the asset folder.
https://github.com/libgdx/libgdx/wiki/Managing-your-assets

rename track.mp3 to something more expressive

FrederikSchlemmer replied:
Audio file renamed and moved to asset folder.

FrederikSchlemmer committed:
Renamed and changed path of music
